dashboards/elasticsearch.indexing.summary.json

```diff
@@ -42,7 +42,7 @@
       }
     }, {
       "id" : "bbf3a7a3-36f2-4787-a242-dc710c597b27",
-      "name" : "Top 5 Slowest Nodes by Flush Time",
+      "name" : "Top 5 Nodes by Most Total Flush Time",
```

Comment:
This metric tracks the total amount of time spent flushing during the cycle, so "Most" sounded more appropriate than "Max". "Max" made it sound more like a per-flush operation statistic like what it was labeled as before.